commands/livecmd.go

```diff
@@ -113,3 +116,21 @@ func GetLiveCommand(name string, f util.Factory) *cobra.Command {
 
 	return liveCmd
 }
+
+func newFactory(cmd *cobra.Command, version string) util.Factory {
```

Comment:
`util.Factory` name is problematic on many levels:
a) What is this Java in 90s? :)
b) `util` itself is a meaningless term. "util for X" should just be called "X", otherwise everything is a util.

Feel free to address this in a separate PR and/or create a package alias to stop the bleeding in the call sites for now.

Reply:
`util.Factory` comes from the kubernetes codebase, so the name is not something we can change (other than creating an import alias).

Reply:
Let's create an alias

Reply:
Renamed `util` to `cluster`